Factory/Headers/computer.h

```diff
@@ -0,0 +1,53 @@
+#ifndef COMPUTER_HPP
+#define COMPUTER_HPP
+
+class IComputer
+{
+public:
+	virtual void getComponentsInfo() = 0;
+}; // IComputer
+
+
+class PC : public IComputer
+{
+public:
+	PC();
+
+public:
+	void getComponentsInfo() override;
```

Comment:
Usually `get*` means that the method should return some value. So would be better to name the method `PrintComponentsInfo` or something like a that.